plotting/matRad_plotDoseSlice.m

```diff
@@ -67,10 +67,17 @@
 cMapScale = size(cMap,1) - 1;
 dose_rgb = ind2rgb(uint8(cMapScale*(dose_slice - window(1))/(window(2)-window(1))),cMap);
 
```

Comment:
This function is very confusing. I would simply put the following


%Use default colormap?
if nargin < 7 || isempty(cMap)
    cMap = jet(64);
end
if nargin < 6 || isempty(alpha)
    alpha = 0.6;
end
if nargin < 8 || isempty(window)
    window = [min(doseCube(:)) max(doseCube(:))];
end
if isempty(threshold)
    threshold = 0;
end

if plane == 1  % Coronal plane
    dose_slice = squeeze(doseCube(slice,:,:));
elseif plane == 2 % sagittal plane
    dose_slice = squeeze(doseCube(:,slice,:));
elseif plane == 3 % Axial plane
    dose_slice = squeeze(doseCube(:,:,slice));
end

cMapScale = size(cMap,1) - 1;
dose_rgb = ind2rgb(uint8(cMapScale*(dose_slice - window(1))/(window(2)-window(1))),cMap);

% plot dose distribution
doseHandle = image('CData',dose_rgb,'Parent',axesHandle);

% set opacities according to windowing
maxDose = max(doseCube(:));
mask = dose_slice < window(2) & dose_slice > max(window(1),threshold*maxDose);
set(doseHandle,'AlphaData', alpha*mask);

end